templates/strapi/files/config/environments/production/request.json

```diff
@@ -0,0 +1,23 @@
+{
+  "session": {
+    "enabled": true,
+    "client": "cookie",
+    "key": "strapi.sid",
+    "prefix": "strapi:sess:",
+    "secretKeys": ["mySecretKey1", "mySecretKey2"],
```

Comment:
If these are something the user needs to change, we should probably call that out in the README.